Zend/tests/bug63882.phpt

```diff
@@ -9,7 +9,7 @@ $testobj2 = new Test;
 $testobj1->x = $testobj1;
 $testobj2->x = $testobj2;
 
-var_dump($testobj1 == $testobj2);
+var_dump($testobj1 <= $testobj2);
```

Comment:
Why did this line need to change?

Reply:
Because for objects, the first == operator is passed to the do_operation handler first, instead of the compare handler. The compare handler is what this test was trying to test, so I updated the test.

I think I made this change prior to making the == operator fall back to the compare handler though, so it's possible this change is no longer necessary.